frontends/krita/krita_diff/client.py

```diff
@@ -267,6 +270,7 @@ def cb(obj):
             self.cfg.set("inpaint_script_list", list(obj["scripts_img2img"].keys()))
             self.cfg.set("face_restorer_model_list", obj["face_restorers"])
             self.cfg.set("sd_model_list", obj["sd_models"])
+            self.cfg.set("sd_vae_list", ["Automatic", "None"] + obj["sd_vaes"])
```

Comment:
Since `async get_state()` already adds the options, there is no need to add it again here. It didn't show up cause the QCombo layout used removes duplicate options. But its harmless anyways.